mlflow/sagemaker/__init__.py

```diff
@@ -218,8 +218,17 @@ def deploy(app_name, model_path, execution_role_arn=None, bucket=None, run_id=No
     :param app_name: Name of the deployed application.
     :param path: Path to the model. Either local if no ``run_id`` or MLflow-relative if ``run_id``
                  is specified.
-    :param execution_role_arn: Amazon execution role with SageMaker rights.
-                               Defaults to the currently-assumed role.
+    :param execution_role_arn: The name of an IAM role granting the SageMaker service permissions to
+                               access the specified Docker image and S3 bucket. If unspecified,
```

Comment:
I would maybe add "S3 bucket containing the model artifacts".

Reply:
Good idea! Added this!